mount/losetup_linux.go

```diff
@@ -84,6 +105,32 @@ func setupLoopDev(backingFile, loopDev string, param LoopParams) (_ *os.File, re
 		}
 	}()
 
+	if ok, err := kernelversion.GreaterEqualThan(
```

Comment:
I think we can use the following pattern to check error.

```
ok, err := kernelversion.GreaterEqualThan(kernelversion.KernelVersion{Kernel: 5, Major: 8})
if err == nil && ok 
```

> kernelversion has cache the kernel version result so we don't need to use sync.Once here.

Reply:
Fixed.